src/system/Form/RadioBoxGroup.js

```diff
@@ -33,31 +33,42 @@ const RadioBoxGroup = React.forwardRef(
 					id={ `o${ index }` }
 					sx={ {
 						p: 3,
-						background: 'none',
```

Comment:
Is this change working as expected? 
They now look gray in the light theme.
[Deploy preview link](https://deploy-preview-167--vip-design-system-components.netlify.app/?path=/story/radioboxgroup--default)

<img width="273" alt="Screenshot RadioBoxGroup - Default ⋅ Storybook 2023-01-23 11-57-47" src="https://user-images.githubusercontent.com/668251/214023544-9d2b090f-f6e4-4380-aa9f-9479738057af.png">

Reply:
@andrea-sdl Check again ;)